<?xml version="1.0"?>
<launch>
    <arg name="model"/>
    <arg name="gui" default="False"/>

    <param name="robot_description"
        textfile="$(find robot_arm)/urdf/robot_arm.urdf"/>
    <param name="use_gui"
        value="$(arg gui)"/>
    
    <node
        name="robot_state_publisher"
        pkg="robot_state_publisher"
        type="state_publisher">
        <remap from="joint_states" to="robotJointState"/>
    </node>
    <node
        name="rviz"
        pkg="rviz"
        type="rviz"
        args="-d $(find robot_arm)/urdf.rviz"
        required="true"/>
</launch>
<!--<launch>
  <arg
    name="model" />
  <arg
    name="gui"
    default="False" />
  <param
    name="robot_description"
    textfile="$(find robot_arm)/urdf/robot_arm.urdf" />
  <param
    name="use_gui"
    value="$(arg gui)" />
  <node
    name="joint_state_publisher"
    pkg="joint_state_publisher"
    type="joint_state_publisher" />
  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="state_publisher" />
  <node
    name="rviz"
    pkg="rviz"
    type="rviz"
    args="-d $(find robot_arm)/urdf.rviz" />
</launch>-->